ui/src/app/pages/profile/profile-registration-step-status.tsx

```diff
@@ -56,42 +59,79 @@ const ProfileRegistrationStepStatus: React.FunctionComponent<Props> =
       incompleteButtonText,
       completedButtonText,
       completeStep,
+      completionTimestamp,
       children
     } = props;
 
-    return (
-      <div style={styles.container}>
-        <div style={styles.title}>
-          { title }
+    if (environment.enableProfileCapsFeatures) {
+      return (
+        <div style={styles.container}>
+          <div style={styles.title}>
+            { title }
+          </div>
+          <FlexColumn>
+            <div>
+              { isComplete ? (
+                <FlexRow>
+                  <ClrIcon style={{color: colors.success, width: '2rem', height: '2rem'}}
+                           shape='check-circle' className='is-solid' />
+                  <FlexColumn style={{marginLeft: '0.5rem'}}>
+                    <div>{ wasBypassed ? 'Bypassed By Admin on:' : completedButtonText + ' on:' }
+                    </div>
+                    <div>{new Date(completionTimestamp).toDateString()}</div>
+                  </FlexColumn>
+                </FlexRow>
+              ) : (
+                <Button
+                  type='purplePrimary'
+                  style={ styles.button }
+                  onClick={ completeStep }
+                >
+                  { incompleteButtonText }
+                </Button>
+              ) }
+            </div>
+            <div style={{marginLeft: '2.5rem'}}>
+              { children }
+            </div>
+          </FlexColumn>
         </div>
-        <div style={{ display: 'flex' }}>
-          <div style={styles.buttonContainer}>
-            { isComplete ? (
-              <Button
+      );
+    } else { // delete this block below once enableProfileCapsFeatures is removed
```

Comment:
TOL: It might be valuable to add the Feature Flag ticket number on all these todo cleanups, so that we have a way to search for all of them.